sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -231,6 +241,32 @@ private [sql] case class CreateTempTableUsing(
   }
 }
 
+private[sql] case class DropTable(
+    tableName: String,
+    isExists: Boolean,
+    temporary: Boolean) extends Command
```

Comment:
yeah, I use a single logical operator here.
If handle the type of the table and then delegate.
Is that ok to do like below? we need catch exceptions in strategy.  If not found I will construct a new MetaStoreRelation for placeholder, it would be nice if you can give me some advice : )
```scala
try {
          val resolvedTable = context.executePlan(UnresolvedRelation(drop.tableIdent)).analyzed
          resolvedTable match {
            case t: MetastoreRelation =>
              ExecutedCommand(DropHiveTable(t, drop.isExists, drop.temporary)) :: Nil
          }
        }
        catch {
          case e: InvalidTableException if drop.isExists =>
            ExecutedCommand(DropHiveTable(MetastoreRelation("","",None, ), drop.isExists, drop.temporary)) :: Nil
        }
     ...
```